.eslintrc.js

```diff
@@ -15,6 +15,13 @@ module.exports = {
       jsx: true,
     },
   },
+
+  settings: {
+    react: {
+      version: require('./package.json').dependencies.react,
```

Comment:
I haven't done this before, but [reading the docs](https://www.npmjs.com/package/eslint-plugin-react#configuration) I notice this other way to get the version too:

```js
{
  "settings": {
    "react": {
      "version": "detect"
    }
  }
}
```

I wonder if the `"detect"` way is simpler?

Reply:
Changed it to detect, and it works well